LICENSE-APACHE

```diff
@@ -198,4 +198,20 @@ Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
-limitations under the License.
\ No newline at end of file
+limitations under the License.
+
+--- LLVM Exceptions to the Apache 2.0 License ----
```

Comment:
@stevenj @Mr-Leshiy Can you point me to the URL where I can read about LLVM license inclusion in OS projects, please?

Reply:
Its clear in the text of the license itself.
This ONLY applies to `wasi-preview1-component-adapter` and `byte-array-literals`  I have asked @Mr-Leshiy to move it.
Basically as it says,  its still Apache2 for that file,  BUT you can distribute a binary you build from it without complying with 4(a), 4(b) and 4(d) above.

The next part doesn't apply to us at all because we are;t linking to GPL code.  But if someone does link just those crates to GPL code they can waive clauses 5 and 9.  But only if a court says so.

Its not viral, it ONLY affects these two forked crates.  It should not be in our normal license.

More than this we would to raise a Legal ticket, but I am 100% comfortable with the slight modifications to Apache2 that these crates have in their license.